### PR TITLE
Fauxembed

### PR DESCRIPTION
**Pull request by kimthompson**

### CODE REVIEW

File: src/atoms/CustomHtml/CustomHtml.js [hunk 1]
```diff
@@ -1,20 +1,67 @@
...
+    isIframe: iframes.length > 0,
+    hasScript: scripts.length > 0,
+    hasFallbackUrl: fallbackUrl || fallbackUrl !== '' || fallbackUrl !== null,
+    isApproved: whitelistRegex.test(fallbackUrl) // TODO: test me in isolation
+  };
+
+  if (!cond.isIframe && cond.hasScript) {
+    // if a valid script is provided but not in an iframe, wrap in an iframe
+    if (cond.hasFallbackUrl && cond.isApproved) {
+      return `<iframe style="over" width="100%" height="650px" frameborder="0" scrolling="yes" marginheight="0" marginwidth="0" src="${fallbackUrl}"></iframe>`;
```
Comment (mattaho) [inline src/atoms/CustomHtml/CustomHtml.js]: I might be wrong, but I don't think the `style="over"` will do anything here

**Comment by mattaho** [comment]
I think for the most part this looks good, I do think this is the sort of thing that really needs tests.

File: src/atoms/CustomHtml/CustomHtml.js (hunk 1 above)
Comment (kimthompson) [reply src/atoms/CustomHtml/CustomHtml.js]: It absolutely won't, and I don't know what that is or how it got in there. Good catch!

**Comment by kimthompson** [comment]
I agree. I just ran out of time. Maybe I'll talk to Peter and see if he can give me a few more hours to get tests set up for this.

**Pull request merged**